configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,1689 @@ benchmark_configs: {
+    "          match_type: ALL_MATCH_ANY_ORDER"
+    "          match_criteria:{"
+    "            filter_regex: \".*sethostname.*setdomainname.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-a\\\\s*always,exit\\\\s*-F\\\\s*arch=b[2346]{2}\\\\s*-S\\\\s+sethostname,setdomainname\\\\s*-k\\\\s+system-locale\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/issue\\\\s+.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/issue\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/issue\\\\.net.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/issue\\\\.net\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/hosts.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/hosts\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/networks.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/networks\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/network/.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/network/\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "          match_criteria:{"
+    "            filter_regex: \".*/etc/netplan/.*-k\\\\s+system-locale.*\""
+    "            expected_regex: \"\\\\s*-w\\\\s*/etc/netplan/\\\\s*-p\\\\s*wa\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+    "          }"
+    "        }"
+    "      }"
+    "  }"
+    "}"
+  }
+}
+
+
+
+benchmark_configs: {
+  id: "ensure-journald-forwardtosyslog-disabled"
+  compliance_note: {
+    title: "Ensure journald ForwardToSyslog is disabled"
+    description: 
+      "Data from journald should be kept within the service and not forwarded to other logging mechanisms."
+    rationale: 
+      "Logs should be handled exclusively by journald to prevent unnecessary forwarding and enhance security."
+    remediation: 
+      "Set the following parameter in the [Journal] section of /etc/systemd/journald.conf or a file in "
+      "/etc/systemd/journald.conf.d/ ending in .conf:\n" 
+      "```\n"
+      "[Journal]\n" 
+      "ForwardToSyslog=no\n\n" 
+      "```\n"
+      "After updating the configuration, reload or restart systemd-journald:\n\n" 
+      "```\n"
+      "# systemctl reload-or-restart systemd-journald\n"
+      "```\n"
+    cis_benchmark: {
+      profile_level: 1
+      severity: MEDIUM
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "    file_checks:{"
```

Comment:
Might be better to replace these two file_checks with one file_check that checks that each ForwardToSyslog=.* is ForwardToSyslog=yes:


```
      "    file_checks:{"
      "      files_to_check:{files_in_dir:{dir_path:\"/etc/systemd/journald.conf.d/\", filename_regex:\".*\\\\.conf\", recursive:false}}"
      "      content_entry:{"
      "        match_type: NONE_MATCH"
      "        match_criteria:{"
      "          filter_regex: \"[^#]*\\\\s*ForwardToSyslog\\\\s*=\\\\s*.*\""
      "          expected_regex: \"[^#]*\\\\s*ForwardToSyslog\\\\s*=\\\\s*yes.*\""
      "        }"
      "      }"
      "    }"

```